parse/src/lockfiles/mod.rs

```diff
@@ -0,0 +1,185 @@
+use std::fmt::Display;
+use std::path::Path;
+use std::str::FromStr;
+
+pub use csharp::CSProj;
+pub use java::{GradleLock, Pom};
+pub use javascript::{PackageLock, YarnLock};
+use phylum_types::types::package::{PackageDescriptor, PackageType};
+pub use python::{PipFile, Poetry, PyRequirements};
+pub use ruby::GemLock;
+use serde::de::IntoDeserializer;
+use serde::{Deserialize, Serialize};
+use strum::{EnumIter, IntoEnumIterator};
+
+mod csharp;
+mod java;
+mod javascript;
+mod parsers;
+mod python;
+mod ruby;
+
+/// A file format that can be parsed.
+#[derive(
+    Clone, Copy, Debug, Deserialize, EnumIter, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize,
+)]
+#[serde(rename_all = "camelCase")]
+#[non_exhaustive]
+pub enum LockFileFormat {
```

Comment:
Lockfile is generally used as a single word in our CLI repository afaik. So I wouldn't capitalize the F here.

Reply:
In this repository, lock file is written as two words 18 times, mostly in user-facing locations. It is written as one word 177 times.

Reply:
Thanks for getting the numbers on that one, I had a hunch but always good to confirm.